### PR TITLE
Test: Refactor health test

### PR DESCRIPTION
**Pull request by eloycoto**
On `healthTest` the use of `jq` command didn't show the full json output,
and it was hard to know the status. Move the execution to use
`cmdRes.Filter` so the health data will be dumped into `test-output.log`

The current output/log that we have is the following:

```
/home/jenkins/workspace/Ginkgo-CI-Tests-Pipeline/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:376
Expected
    <string>: "Connection timed out"

to contain substring
    <string>: null
/home/jenkins/workspace/Ginkgo-CI-Tests-Pipeline/src/github.com/cilium/cilium/test/k8sT/Health.go:109
```

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

The output of the jsonpath: 

```
vagrant@k8s1:~$ kubectl -n kube-system exec -ti cilium-lpqg4 -- cilium-health status -o jsonpath='{.nodes[1].endpoint.http.status}'
Connection timed out
vagrant@k8s1:~$ kubectl -n kube-system exec -ti cilium-lpqg4 -- cilium-health status -o jsonpath='{.nodes[1].host.primary-address.icmp.status}'
Connection timed out
vagrant@k8s1:~$
```

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/4661)

### CODE REVIEW

**Comment by eloycoto** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

File: test/k8sT/Health.go [hunk 1]
```diff
@@ -97,17 +97,19 @@ var _ = Describe(testName, func() {
 		apiPaths := []string{
 			"endpoint.icmp",
 			"endpoint.http",
-			"host.\"primary-address\".icmp",
-			"host.\"primary-address\".http",
+			"host.primary-address.icmp",
+			"host.primary-address.http",
 		}
 		for node := 0; node <= 1; node++ {
+			healthCmd := fmt.Sprintf("cilium-health status -o json")
```
Comment (tgraf) [inline test/k8sT/Health.go]: Sprintf is unnecessary

**Review by tgraf — CHANGES_REQUESTED** [review]

**Comment by eloycoto** [comment]
test-me-please

File: test/k8sT/Health.go (hunk 1 above)
Comment (eloycoto) [reply test/k8sT/Health.go]: fixed.

**Review by tgraf — APPROVED** [review]

**Pull request merged**